_includes/icons/gaming.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?><!-- Uploaded to: SVG Repo, www.svgrepo.com, Generator: SVG Repo Mixer Tools -->
```

Comment:
We tend to use remix icons for this, how about using them? https://remixicon.com/

Reply:
Okay